front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -97,6 +104,10 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
     useAddProjectFolder();
   const { mutate: updateProjectFolder } = useUpdateProjectFolder();
 
+  const showDescriptionBuilder = useFeatureFlag({
+    name: 'project_description_builder',
+  });
```

Comment:
Just want to confirm the Folder Description Builder should indeed follow the same FF? If so, would it be simple to re-name the FF to just description_builder? :thinking:

Reply:
Yes I don't see any reason to use another one. But feature flags are hard to rename without adding some kind of data migration for them. Maybe I could do that in a separate PR.